peribolos/knative-sandbox.yaml

```diff
@@ -565,8 +565,8 @@ orgs:
           Knative Release Leads:
             description: Active member of the Knative Release Leads rotation.
             maintainers:
-            - julz
-            - n3wscott
```

Comment:
good night, sweet access

Reply:
you mean we don't get to keep it till January 20th? 😢